NtupleProducer/plugins/NtupleProducer.cc

```diff
@@ -359,10 +360,11 @@ NtupleProducer::produce(edm::Event& iEvent, const edm::EventSetup& iSetup) {
             return c.et; 
         }
   } );
+  */
   //// ---- Calibration from Phil's workflow
-  //ecalClusterer_.correct( [](const l1pf_calo::Cluster &c, int ieta, int iphi) -> double { 
-  //      return ecorrector_->correct(0., c.et, ieta, iphi); // FIXME is ieta or abs(ieta) ?
-  //} );
+  ecalClusterer_.correct( [=](const l1pf_calo::Cluster &c, int ieta, int iphi) -> double { 
+      return ecorrector_->correct(0., c.et, ieta, iphi); // FIXME is ieta or abs(ieta) ? => its ieta
```

Comment:
To me photons don't get corrections unless I change https://github.com/violatingcp/FastPUPPI/blob/91X_calib/NtupleProducer/src/corrector.cc#L63 to `if(fFrac > 0.8 && fNFrac > 1) return fTotal;` , otherwise the ecal-only corrector always returns the uncorrected enery
After that,  photons look ok (especially up to eta 3) but pions have linking issues below eta 1.5
https://gpetrucc.web.cern.ch/gpetrucc/drop/plots/l1stage2/910pre2/t6_PU0/?match=photon*-l1pf
https://gpetrucc.web.cern.ch/gpetrucc/drop/plots/l1stage2/910pre2/t6_PU0/pion_eta_00_15-l1pf_pthighest.png